grouper/ctl/sync_db.py

```diff
@@ -17,10 +18,36 @@ def sync_db_command(args):
 
     # Add some basic database structures we know we will need if they don't exist.
     session = make_session()
+    # This group is needed to bootstrap a Grouper installation.
+    admin_group = Group.get(session, name="grouper-administrators")
+    if not admin_group:
+        admin_group = Group(
+                groupname="grouper-administrators",
+                description="Administrators of the Grouper system.",
+                canjoin="nobody",
+        )
+
+        try:
+            admin_group.add(session)
+            session.flush()
+        except IntegrityError:
+            session.rollback()
+            raise Exception('Failed to create group: grouper-administrators')
+    admin_group_permission_names = [perm[1] for perm in admin_group.my_permissions()]
+
     for name, description in SYSTEM_PERMISSIONS:
-        test = Permission.get(session, name)
-        if test:
+        permission = Permission.get(session, name)
+        if permission and name not in admin_group_permission_names:
+            try:
+                admin_group.grant_permission(permission)
+                session.commit()
+            except IntegrityError:
+                # The permission is already extant in the group, carry on.
+                session.rollback()
+            continue
+        elif permission:
             continue
```

Comment:
May be worth adding a quick test to verify `sync_db` creates the group and adds the correct permissions.